src/configs/index.ts

```diff
@@ -1,7 +1,9 @@
 export const config = {
     rolesToTag: ["976139361903067190", "976141715197009920"],
     supportTicketChannels: {
-        from: "977000089161699360",
-        to: "977000110670094347",
+        from: "979553780922195978",
+        to: "979552193835331654",
     },
+    rolesAllowedToUse: ["976141715197009920", "976139361903067190"],
+    errorMessage: 'You do not have the required role to use this feature. Check out https://www.patreon.com/stuyk'
```

Comment:
Why is this a config value? The message rarely changes - Therefore it's not required